app/controllers/promos_controller.rb

```diff
@@ -9,15 +9,22 @@ def new
 
   def create
     @promo = Promo.new(promo_model_params)
+    @promo.employee_id = current_employee.id
```

Comment:
```suggestion
    @promo = current_employee.build_promo(promo_model_params)
```

Dá pra usar da amarração pra já criar amarrado